spec/classes/init_spec.rb

```diff
@@ -672,4 +672,34 @@
       end
     end
   end
+
+  describe 'with disable_monitor set' do
+    let(:facts) { { :osfamily => 'RedHat' } }
+
+    [true,'true',false,'false'].each do |value|
+      context "to #{value} as #{value.class}" do
+        let(:params) { { :disable_monitor => value } }
+
+        it { should compile.with_all_deps }
+        it { should contain_class('ntp') }
+
+        if value == true or value == 'true'
+          it { should contain_file('ntp_conf').with_content(/^disable monitor$/) }
+        else
```

Comment:
Given our conversation you could just put

``` ruby
if value == true
```

and not worry about `'true'` because the module should always make it boolean `true`.